src/components/SearchResultsList/SearchResultsList.js

```diff
@@ -53,25 +60,72 @@ const SearchResultsList = ({
     </Layout>
   );
 
-  return (
-    <MultiColumnList
-      visibleColumns={['isSelected', 'name', 'selectedCount', 'titleCount']}
-      columnMapping={{
-        isSelected: <FormattedMessage id="ui-plugin-find-package-title.resultsPane.status" />,
-        name: <FormattedMessage id="ui-plugin-find-package-title.resultsPane.name" />,
-        selectedCount: <FormattedMessage id="ui-plugin-find-package-title.resultsPane.titlesSelected" />,
-        titleCount: <FormattedMessage id="ui-plugin-find-package-title.resultsPane.totalTitles" />,
-      }}
-      columnWidths={{
+  const getVisibleColumns = () => {
+    let visibleColumns = ['isSelected', 'name', 'selectedCount', 'titleCount'];
+
+    if (isMultiSelect) {
+      visibleColumns = ['checked', ...visibleColumns];
+    }
+
+    return visibleColumns;
+  };
+
+  const getColumnMapping = () => {
+    let columnMapping = {
+      isSelected: <FormattedMessage id="ui-plugin-find-package-title.resultsPane.status" />,
+      name: <FormattedMessage id="ui-plugin-find-package-title.resultsPane.name" />,
+      selectedCount: <FormattedMessage id="ui-plugin-find-package-title.resultsPane.titlesSelected" />,
+      titleCount: <FormattedMessage id="ui-plugin-find-package-title.resultsPane.totalTitles" />,
```

Comment:
Please, set values as `intl.formatMessage({ id: '' })` instead of `<FormattedMessage id='' />` for columnMapping in MCL. This point can help to set correct arialabel